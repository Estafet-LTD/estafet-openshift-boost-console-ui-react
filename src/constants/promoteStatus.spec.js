import PROMOTE_STATUS from './promoteStatus';

describe('Constants: PROMOTE_STATUS', () => {
  it('should match the snapshot', () => {
    expect(PROMOTE_STATUS).toMatchSnapshot();
  });
});
