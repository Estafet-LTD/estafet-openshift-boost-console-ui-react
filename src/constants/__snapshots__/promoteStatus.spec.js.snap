// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Constants: PROMOTE_STATUS should match the snapshot 1`] = `
Object {
  "FULLY_PROMOTED": "FULLY_PROMOTED",
  "NOT_PROMOTED": "NOT_PROMOTED",
  "PARTIALLY_PROMOTED": "PARTIALLY_PROMOTED",
}
`;
